dockerfiles/che/launcher.sh

```diff
@@ -0,0 +1,380 @@
+#!/bin/sh
+
+init_logging() {
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  RED='\033[0;31m'
+  NC='\033[0m'
+}
+
+init_global_variables() {
+
+  CHE_SERVER_CONTAINER_NAME="che-server"
+  CHE_SERVER_IMAGE_NAME="codenvy/che"
+  CHE_LAUNCHER_IMAGE_NAME="codenvy/che-launcher"
+
+  # Possible Docker install types are:
+  #     native, boot2docker or moby
+  DOCKER_INSTALL_TYPE=$(get_docker_install_type)
+
+  # User configurable variables
+  DEFAULT_DOCKER_HOST_IP=$(get_docker_host_ip)
```

Comment:
yes I agree
Also two dockerfiles have the same parent images and install docker in it by duplicating lines
I also think this common stuff should be added in a shared image so that we do not duplicate these layers